webrtc/videoroom/lib/videoroom/room.ex

```diff
@@ -84,35 +85,29 @@ defmodule Videoroom.Room do
       end
 
     endpoint = %WebRTC{
-      ice_name: peer_id,
+      ice_name: peer.id,
       extensions: %{},
       owner: self(),
       stun_servers: state.network_options[:stun_servers] || [],
       turn_servers: state.network_options[:turn_servers] || [],
       handshake_opts: handshake_opts,
-      log_metadata: [peer_id: peer_id],
-      filter_codecs: fn {rtp, fmtp} ->
-        case rtp.encoding do
-          "opus" -> true
-          "H264" -> fmtp.profile_level_id === 0x42E01F
-          _unsupported_codec -> false
-        end
-      end
+      log_metadata: [peer_id: peer.id]
     }
 
-    Engine.accept_peer(rtc_engine, peer_id)
+    Engine.accept_peer(rtc_engine, peer.id)
 
     :ok =
       Engine.add_endpoint(rtc_engine, endpoint,
-        peer_id: peer_id,
+        peer_id: peer.id,
         node: peer_node
       )
 
     {:noreply, state}
   end
 
   @impl true
-  def handle_info({_rtc_engine, {:peer_left, _peer_id}}, state) do
+  def handle_info(%Message.PeerLeft{peer: peer}, state) do
+    Membrane.Logger.info("Peer #{inspect(peer.id)} left RTC Engine")
```

Comment:
Do we need this, as we don't log messages about new peers joining the room?

Reply:
I added log for new peer. It can be helpful to see when people are joining and leaving :slightly_smiling_face: